app/src/main/res/values/striga_strings.xml

```diff
@@ -29,40 +29,34 @@
     <string name="striga_exit_sms_input_warning_dialog_message">You can confirm the phone number and finish the registration later</string>
     <string name="striga_exit_sms_input_warning_dialog_btn_positive">Yes, leave the page</string>
     <string name="striga_exit_sms_input_warning_dialog_btn_negative">No, continue</string>
-    <string name="striga_enter_personal_data_info">Enter your personal data to open an account. Please, use your real credentials.</string>
-    <string name="striga_contacts">Contacts</string>
-    <string name="striga_email_hint">prefilled@gmail.com</string>
-    <string name="striga_email">Email</string>
-    <string name="striga_phone_number">Phone number</string>
-    <string name="striga_phone_number_hint">+*user number w3*</string>
-    <string name="striga_credentials">Credentials</string>
-    <string name="striga_first_name">First name</string>
-    <string name="striga_surname">Surname</string>
-    <string name="striga_enter_hint">Enter</string>
-    <string name="striga_date_of_birth">Date of Birth</string>
-    <string name="striga_date_of_birth_hint">DD.MM.YYYY</string>
-    <string name="striga_country_of_birth">Country of birth</string>
-    <string name="striga_country_of_birth_hint">Select from list</string>
+    <string name="striga_form_personal_info_label">Personal Information</string>
```

Comment:
maybe striga_sign_up instead of form?

Reply:
it's form section title, not just abstract sign up, maybe better to call it `striga_form_personal_info_section_title`, wdyt?